reactpodcast/the-react-podcast-19.md

```diff
@@ -0,0 +1,6 @@
+* [Shadowbox.js](http://www.shadowbox-js.com/) --- Michael's first open source project
+* [React Router](https://reacttraining.com/react-router/)
+* [UNPKG](https://unpkg.com/) --- A CDN for NPM
+* [UNPKG: The CDN for everything on npm](https://www.youtube.com/watch?v=2rhkgB8Cohc) --- Michael's 2017 React Rally talk on web modules
```

Comment:
This one was missing on https://reactpodcast.simplecast.fm/19 but found the video and added the link.